client/src/hooks/courseManagement.js

```diff
@@ -0,0 +1,77 @@
+import { useCallback } from 'react';
+
+import useGrpc from './grpc';
+
+import { CourseManagementClient } from '../proto/content-management_grpc_web_pb';
```

Comment:
you should put 'from...' on a new line to not reach characters limit

Reply:
The problem is in prettier. Next branch will be for refactoring and I will fix this